conans/client/generators/cmake_find_package_common.py

```diff
@@ -92,18 +92,17 @@ def find_transitive_dependencies(public_deps_names, find_modules):
         find = textwrap.dedent("""
             if(NOT {dep_filename}_FOUND)
                 if(${{CMAKE_VERSION}} VERSION_LESS "3.9.0")
-                    find_package({dep_filename} REQUIRED NO_MODULE)
+                    find_package({dep_name} REQUIRED NO_MODULE)
```

Comment:
I don't understand why here it is not ``dep_filename``. Seems like variable naming is very confusing

Reply:
You are right. For a moment I thought this macro was used by `cmake` and `cmake_multi` generators. I will change the naming. Thanks

Reply:
Good. Fortunately, it seems the tests were failing: https://conan-ci.jfrog.info/blue/organizations/jenkins/ConanTestSuite/detail/PR-7447/3/pipeline/181. Hopefully everything is covered well enough by testing.